client/src/components/Header/Navbar.tsx

```
@@ -28,153 +28,158 @@
+                  key={link.name}
+                  href={link.link}
+                  onClick={() => handleNavClick(link.name)}
+                  className={`relative text-gray-700 ${
+                    activeLink === link.name
+                      ? "text-green-600 font-semibold"
+                      : "hover:text-green-600"
+                  }`}
+                >
+                  {link.label}
+                  {activeLink === link.name && (
+                    <span className="absolute -bottom-2 left-0 right-0 h-1 bg-green-600 rounded-full"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)